docs/body.md

```diff
@@ -390,32 +407,71 @@ Instances
 
 The following device level metrics are collected:
 
-* Flavors
-* Total (count) Images
+#### Flavors
+
 * Total (count)
-* Total count per image state Servers
+
+#### Images
+
+* Total (count)
+* Total count per image state
+
+#### Servers
+
 * Total (count)
 * Total count per server state
-* Queues
+
+#### Queues
+
 * Event (count)
-* Performance (count) Agents
+* Performance (count)
+
+#### Agents
+
 * Total (count)
-* Total count per agent type Networks
+* Total count per agent type
+
+#### Networks
+
 * Total (count)
-* Total count per network state Routers
+* Total count per network state
+
+#### Routers
+
 * Total (count)
-* Total count per router state Volumes
+* Total count per router state
+
+#### Volumes
+
 * Total (count)
-* Total count per volume state Volume Snapshots
+* Total count per volume state
+
+#### Volume Snapshots
+
 * Total (count)
-* Total count per volume snapshot state Volume Pool
+* Total count per volume snapshot state
+
+#### Volume Pool
+
 * Total (count)
 * Total count per volume pool state
 
-Note: All events processed through Ceilometer are automatically 
-exposed via the Zenoss Event Console, and all metrics collected by Ceilometer
-may be collected and graphed in Zenoss through the use of custom monitoring 
-templates.
+#### Notes:
+
+* All events processed through Ceilometer are automatically 
```

Comment:
The first part of this is no longer true, i think i mentioned in a different PR that we need to also fix that.  If you want to do that part separately, that's fine, and this is otherwise approved